server/src/models/questions.models.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+
+const questionsSchema = new mongoose.Schema({
+
+    questionText: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    tags: {
+        type: String,
+        required: true,
+        trim: true
+    },
+    createdBy: {
+        type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true
```

Comment:
Please write the object's element in a separate line or you can formate code for more readability.

Reply:
okay